client/src/pages/ErrorPage.tsx

```diff
@@ -0,0 +1,17 @@
+import { Stack, Typography } from "@mui/material"
+import { Link, useRouteError } from "react-router-dom"
+
+export const ErrorPage = () => {
+  const error:any = useRouteError();
```

Comment:
Don't need to add the type annotation.

Reply:
Does this work as well? Looking at the documentation it says it only works with the data router, which we are not using.